pkg/jobrunaggregator/jobrunbigqueryloader/types.go

```diff
@@ -46,8 +47,12 @@ func (d dryRunInserter) Put(ctx context.Context, src interface{}) (err error) {
 	buf := &bytes.Buffer{}
 	fmt.Fprintf(buf, "BULK INSERT into %v\n", d.table)
 	for i := 0; i < srcVal.Len(); i++ {
-		s := srcVal.Index(i).Interface()
-		fmt.Fprintf(buf, "\tINSERT into %v: %v\n", d.table, s)
+		s := srcVal.Index(i).Interface().(*testRunRow)
```

Comment:
I would rather see this structured as a registry of types that can added to the dryrunInserter instance.